pages/declaration.js

```diff
@@ -135,7 +135,7 @@ class Declaration extends React.Component {
             </Checkbox>
             <Paragraph linkRenderer={NewTabLinkRenderer}>
               {this.props.t(
-                "[Details on how we use your data](/pdfs/feedback)"
+                "[How we will use your data (feedback pdf opens in new window)](/pdfs/feedback)"
```

Comment:
I know this is what the report suggestion was, but I don't think "feedback pdf" really makes sense from a user's perspective.  Maybe just say "(PDF opens in a new window)"?
Update common.json too.